.gitattributes

```diff
@@ -7,3 +7,7 @@
 # Have Github ignore js vendored files.
 # https://github.com/gramps-project/gramps/tree/master/data/javascript
 #data/javascript/*.js linguist-vendored
+
+# don't mess with line endings for Gedcom files
+*.ged binary

```

Comment:
`binary` disable diffing, too. Are you sure that's what you want? Maybe you want to unset `text` to disable end-of-line normalization.